equinox/internal/_omega.py

```diff
@@ -29,13 +31,22 @@ class _ω(metaclass=_Metaω):
         ```
 
         This is entirely equivalent to the above.
+
+
+        If `structure` is passed then
+        ```
+        ω(a,structure=b).call(fn) = jax.tree_util.tree_map(lambda x,y: fn(y), b, a)
+        ```
+        where b is a prefix PyTree of a. This evaluates a at the nodes corresponding
```

Comment:
Nits: add backticks around variable names. Add spaces to code block to format it like Black would. Rename `x` to `_`.